resources/assets/coffee/react/beatmapset-page/stats.coffee

```diff
@@ -79,7 +79,12 @@ export class Stats extends React.Component
       div className: 'beatmapset-stats__row beatmapset-stats__row--advanced',
         table className: 'beatmap-stats-table',
           tbody null,
-            for stat in ['cs', 'drain', 'accuracy', 'ar', 'stars']
+            stats = 
+              if @props.beatmap.mode == 'taiko'
+                ['drain', 'accuracy', 'stars']
+              else
+                ['cs', 'drain', 'accuracy', 'ar', 'stars']
```

Comment:
This renders the array into the element body, ti should be set somewhere where the return value doesn't render

Reply:
fixed